lib/src/main/java/com/samsung/android/sdk/penremote/SpenEvent.kt

```diff
@@ -47,9 +47,28 @@ import kotlinx.parcelize.Parcelize
  *
  * @hide
  */
-@Parcelize
 data class SpenEvent(val timeStamp: Long, val values: Array<Float>) : Parcelable {
-    init {
-        TODO("Stop using Parcelize and implement Parcelable manually; remove the library reference to kotlin-parcelize")
+    constructor(parcel: Parcel) : this(
+        parcel.readLong(),
+        // TODO: Might not be correct, but works for now
+        Array(parcel.readInt() + 1) { parcel.readFloat() }
```

Comment:
It's not, but this will work. Leave the TODO in, as it means `AirMotionEvent/ButtonEvent` are off by one